### ISSUE TITLE
Why the validation error is lower than training error

### ISSUE THREAD

**Issue by VikrantKarale** [ruotianluo/pytorch-faster-rcnn#116]
I have modified a original code so that I can plot training vs validation loss curves for the losses obtained after each epoch. But, I am getting validation loss less than training loss. I am plotting average training and validation loss (total loss divided by total images). The plot is shown below:
![error_adam_1e-8](https://user-images.githubusercontent.com/30911832/47805731-0710b580-dd5e-11e8-87e6-1b68383138b5.png)
Can anybody tell why it might be happening?

**Comment by SerajRzmi** [comment]
in my opinion this is not overfitting but I have some recommends if you use NN: 
try and test these:
1. don't use drop out in primary layers 
2. use low drop out or regularization(L1,L2) rate
